mars/services/scheduling/supervisor/globalslot.py

```diff
@@ -20,25 +20,24 @@
 
 from .... import oscar as mo
 from ....typing import BandType
+from ... import Resource, ZeroResource
 
 logger = logging.getLogger(__name__)
 
 
 class GlobalSlotManagerActor(mo.Actor):
```

Comment:
slot -> Resource in this pr, should the class `GlobalSlotManagerActor`  be changed to `GlobalResourceManagerActor`?

Reply:
Good suggestion, this file should be renamed to `globalresource.py` too.